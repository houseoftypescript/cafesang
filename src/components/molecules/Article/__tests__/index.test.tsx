import { render } from '@testing-library/react';
import { Article } from '../index';

describe('Button', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Article url={''} title={''} image={''} category={''} description={''} />
    );
    expect(container).toMatchSnapshot();
  });
});
