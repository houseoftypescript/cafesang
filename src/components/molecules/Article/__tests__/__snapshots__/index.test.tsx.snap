// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button to match snapshot 1`] = `
<div>
  <a
    href=""
    target="_blank"
  >
    <div
      class="grid grid-cols-1 md:grid-cols-2 items-center gap-4 md:gap-8"
    >
      <div
        class="col-span-1"
      >
        <div
          class="pb-[75%] bg-center bg-cover bg-no-repeat rounded"
          style="background-image: url();"
          title=""
        />
      </div>
      <div
        class="col-span-1"
      >
        <div
          class="flex flex-col gap-2"
        >
          <p
            class="text-red-500 font-bold"
          />
          <h3
            class="truncate-multipleline text-xl font-medium"
          />
          <p
            class="truncate-multipleline"
          />
        </div>
      </div>
    </div>
  </a>
</div>
`;
